src/lib/autoinstall/presenters/partition.rb

```diff
@@ -84,6 +84,17 @@ def drive_type
         drive_presenter.type
       end
 
+      # String representation of #fstab_options
+      #
+      # The field is actually called fstopt in the AutoYaST xml and treated like a plain string,
+      # so this method provides a more direct communication between the profile and the current
+      # UI (which also uses a plain InputField).
+      #
+      # @return [String]
+      def fstopt
+        fstab_options&.join(",")
```

Comment:
NP: Can *fstab_options* be nil? If so, I would avoid to return nil, but an empty string. If not, I would not use `&.`.

Reply:
Yes, it can be nil. The point is that the classes at `Y2Storage::AutoinstProfile` return nil if the attribute is omitted in the XML and I think this should behave the same instead of turning the nil into an empty value (it's a decorator of one of those classes, after all).

Reply:
Ok then. Please, adapt the return value documentation to indicate nil too.